forgevtt-module.js

```diff
@@ -2074,11 +2121,11 @@ class ForgeVTT_FilePicker extends FilePicker {
             const flip = url.searchParams.get('flip') === "true";
             const flop = url.searchParams.get('flop') === "true";
             const blur = parseInt(url.searchParams.get('blur')) || 0;
-            options.find('input[name="no-optimizer"]').prop('checked', noOptimizer);
-            options.find('input[name="flip"]').prop('checked', flip);
-            options.find('input[name="flop"]').prop('checked', flop);
-            options.find('select[name="blur"]').val(blur);
-            options.show();
+            ForgeVTT.ensureIsJQuery(options).find('input[name="no-optimizer"]').prop('checked', noOptimizer);
+            ForgeVTT.ensureIsJQuery(options).find('input[name="flip"]').prop('checked', flip);
+            ForgeVTT.ensureIsJQuery(options).find('input[name="flop"]').prop('checked', flop);
+            ForgeVTT.ensureIsJQuery(options).find('select[name="blur"]').val(blur);
+            ForgeVTT.ensureIsJQuery(options).show();
```

Comment:
Looks like it would be worth reassigning `options` at the top of the method.
```js
    _onInputChange(options, input) {
        options = ForgeVTT.ensureIsJQuery(options);
        ...
```

Reply:
eslint cautions against param reassignment but I agree with the idea.
I assigned to a new var `_options` as a middle ground ensuring we don't run into any weird behaviours due to reassignment
```js
    _onInputChange(options, input) {
        // New var _options to avoid param reassignment, and ensure we're working with a jQuery object
        const _options = ForgeVTT.ensureIsJQuery(options);
```